doc/frequently-asked-questions.rst

```diff
@@ -488,7 +488,7 @@ The solution is to add it as another package to the environment:
 
 .. code:: shell
 
-   $ nix-shell -p '(haskellPackages.ghcWithPackages.override { useLLVM = true; }) (ps: with ps; [mtl])' 'haskellPackages.llvmPackages.llvm'
+   $ nix-shell -p '(haskellPackages.ghcWithPackages.override { useLLVM = true; }) (ps: with ps; [mtl])' 'haskellPackages.ghc.llvmPackages.llvm'
```

Comment:
This is fair since it is about maknig LLVM tools matching the LLVM backend available.